idea_town/frontend/static-src/app/templates/experiment-page.js

```diff
@@ -6,9 +6,9 @@ export default `
         <div class="details-header">
           <h1 data-hook="title"></h1>
           <div class="idea-controls">
-            <button data-hook="feedback" id="feedback-button" class="button primary">Give Feedback</button>
-            <button data-hook="uninstall" id="uninstall-button" class="button neutral"><span class="state-change-inner"></span><span class="transition-text">Disabling...</span><span class="default-text">Disable <span data-hook="title"></span></span></button>
-            <button data-hook="install" id="install-button" class="button primary"><span class="state-change-inner"></span><span class="transition-text">Enabling...</span><span class="default-text">Enable <span data-hook="title"></span></span></button>
+            <button data-l10n-id="giveFeedback" data-hook="feedback" id="feedback-button" class="button primary">Give Feedback</button>
+            <button data-hook="uninstall" id="uninstall-button" class="button neutral"><span class="state-change-inner"></span><span data-l10n-id="disableExperimentTransition" class="transition-text">Disabling...</span><span data-l10n-id="disableExperiment" class="default-text">Disable <span data-hook="title"></span></span></button>
```

Comment:
> If we're intending to localize the individual experiment names, will this part (scrolled off the screen) work?

Localizing content from the database - as opposed to app templates - isn't covered in this PR. I should add that as a note to the PR. Not sure how we're going to do that yet, but it will probably happen at the Django server level rather than the Ampersand client level.

Would be nice if DB content like experiment names could be localized through the same interface that the app itself uses, but I can't remember a Mozilla project where things worked out that way.

Of course, this PR doesn't cover a localization interface, either. Kind of thinking Pontoon, but I don't have any experience with it yet.

> Unfortunately, the l20n.js docs don't mention nested html and localized text, so it's not clear what the recommended approach would be.

[In the localized strings](https://github.com/mozilla/idea-town/pull/372/files#diff-4eb6e773dc567e34f0afea71a400cb5fR37), I didn't include the `<span>` and punted like so:

    <disableExperiment "Disable {{title}}">

The only reason the `<span>` is there, anyway, is for Ampersand to have something as a hook for the experiment title. L20N takes care of that as part of the localization anyway. Not that that solves the general problem of markup in l10n strings, but I don't think it's an issue in this particular case.